tests/e2e/tests/test_rds_s3.py

```diff
@@ -266,13 +266,15 @@ def setup(self, metadata, port_forward, cluster, region, delete_s3_bucket_conten
         # Disable caching in KFP
         # By default KFP will cache previous pipeline runs and subsequent runs will skip cached steps
         # This prevents artifacts from being uploaded to s3 for subsequent runs
-        patch_body = unmarshal_yaml(DISABLE_PIPELINE_CACHING_PATCH_FILE)
-        k8s_admission_registration_api_client = (
-            create_k8s_admission_registration_api_client(cluster, region)
-        )
-        k8s_admission_registration_api_client.patch_mutating_webhook_configuration(
-            "cache-webhook-kubeflow", patch_body
-        )
+
```

Comment:
I am not sure if we should do this. 
@akartsky do u have opinion?

Reply:
@akartsky can answer as well but I think this is fine since the webhook is unable to be created currently
```
$ kubectl get MutatingWebhookConfiguration cache-webhook-kubeflow
Error from server (NotFound): mutatingwebhookconfigurations.admissionregistration.k8s.io "cache-webhook-kubeflow" not found

$ kubectl get MutatingWebhookConfiguration
NAME                                               WEBHOOKS   AGE
admission-webhook-mutating-webhook-configuration   1          143m
cert-manager-webhook                               1          143m
inferenceservice.serving.kserve.io                 2          143m
inferenceservice.serving.kubeflow.org              3          143m
istio-sidecar-injector                             4          143m
katib.kubeflow.org                                 2          143m
pod-identity-webhook                               1          165m
sinkbindings.webhook.sources.knative.dev           1          143m
vpc-resource-mutating-webhook                      1          165m
webhook.eventing.knative.dev                       1          143m
webhook.istio.networking.internal.knative.dev      1          143m
webhook.serving.knative.dev                        1          143m
```

Reply:
should we still do this given that Kartik has a patch/fix.